nalito223/hotel
# Feature/html css skeleton

### Summary: 
This branch includes a starting point for all the html elements and css styling for the site that I will be building in this project. NOTE: this is a static site without any Javascript functionality. 

### How to test:
Pull down the branch and run `NPM start`. Then, explore the site and test if buttons are reactive and that the layout is easily understood while accommodating the user stories. Test the scrolling feature in the table view. Take a look at the CSS and HTML files to see if there is any organizational or syntax issues.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/110144802/200887399-23bb9a52-dc84-4dd3-bc55-d65c7283cd2e.png">
